AElf.Kernel/BlockHeader.cs

```diff
@@ -42,16 +36,32 @@ public BlockHeader(IHash<IBlock> preBlockHash)
         /// </summary>
         public int Nonce { get; set; }
 
+        #region Private fields
+        private MerkleTree<ITransaction> TransactionTrie = new MerkleTree<ITransaction>();
```

Comment:
private filed should be _transactionTrie, or _merkleTreeOfTransactions

Reply:
This is purely coding convention, but I agree with it so I will put it in the style guide.